mlir/lib/Target/KokkosCpp/TranslateToKokkosCpp.cpp

```diff
@@ -2425,6 +2568,12 @@ LogicalResult KokkosCppEmitter::emitTupleType(Location loc, ArrayRef<Type> types
 
 //Version for when we are just emitting C++
 LogicalResult emitc::translateToKokkosCpp(Operation *op, raw_ostream &os, bool declareVariablesAtTop) {
+  /*
```

Comment:
Note: this code here lets you attach a debugger before emitting ops, even if running through Python. Will add a permanent way to enable this later (maybe an environment variable, maybe a ``bool pauseForDebugging`` option)